docs/newmode.rst

```diff
@@ -1,7 +1,7 @@
 New/Mode
 ==========
 
-`New/Mode <https://www.newmode.net/>` is the multi-channel advocacy & civic engagement platform
+`New/Mode <https://www.newmode.net/>`_ is the multi-channel advocacy & civic engagement platform
```

Comment:
Do you think it'd be worth modifying the copy here to be less like marketing speak? Most of the other connectors have very dry, matter-of-fact descriptions of the products they're connecting to, whereas this one seems to be a copy/pasted summary from the vendor.

Reply:
Fixed.